Doc/library/pickle.rst

```diff
@@ -511,6 +511,30 @@ The :mod:`pickle` module exports three classes, :class:`Pickler`,
       Release the underlying buffer exposed by the PickleBuffer object.
 
 
+.. _pickle-cli:
+
+Command-line interface
+----------------------
+
+The :mod:`pickle` module can be invoked as a script from the command line,
+it will display contents of the pickle files. However, when the pickle file
+that you want to examine comes from an untrusted source, ``-m pickletools``
+is a safer option because it does not execute pickle bytecode, see
+:ref:`pickletools CLI usage <pickletools-cli>`.
+
+.. code-block:: bash
+
+   python -m pickle [pickle_file ...]
+
+The following options are accepted:
+
+.. program:: pickle
+
+.. option:: pickle_file
+
+   The pickle file(s).
```

Comment:
```suggestion
   The pickle file.
```

Reply:
Maybe "A pickle file" instead? otherwise it may hint that there is only one expected file.

Reply:
```suggestion
   A pickle file.
```